tests/test_e2e_10_mef_eline.py

```diff
@@ -613,3 +693,649 @@ def test_080_create_and_remove_ten_circuits_ten_times(self):
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:3"}}
+            ]
+        }
+        response = requests.post(api_url, data=json.dumps(payload1),
+                                 headers={'Content-type': 'application/json'})
+        data = response.json()
+        evc1 = data['circuit_id']
+        time.sleep(10)
+        payload2 = {
+            "primary_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:4"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:3"}}
+            ]
+        }
+        # It sets a new circuit's primary_path
+        response = requests.patch(api_url + evc1, data=json.dumps(payload2),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 200
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['primary_path'][0]['endpoint_a']['id'] == payload2['primary_path'][0]['endpoint_a']['id']
+        assert data['primary_path'][0]['endpoint_b']['id'] == payload2['primary_path'][0]['endpoint_b']['id']
+        assert data['active'] is True
+
+    def test_140_patch_backup_path(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        payload = {
+            "name": "my evc1",
+            "enabled": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+                "tag": {
+                    "tag_type": 1,
+                    "value": 101
+                }
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:03:1",
+                "tag": {
+                    "tag_type": 1,
+                    "value": 101
+                }
+            },
+            "primary_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:2"}},
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:02:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:2"}}
+            ],
+            "backup_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:4"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:3"}}
+            ]
+        }
+        response = requests.post(api_url, data=json.dumps(payload),
+                                 headers={'Content-type': 'application/json'})
+        data = response.json()
+        evc1 = data['circuit_id']
+
+        time.sleep(10)
+
+        payload = {
+            "backup_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:2"}}
+            ]
+        }
+
+        # It sets a new circuit's backup_path
+        requests.patch(api_url + evc1, data=json.dumps(payload),
```

Comment:
this should return 400 and should have an assertion on the return_code

Reply:
Resolved with commit [baa70d4](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/baa70d4621436584b34a98c08535132fb4653be6)